tests/deployment/test_autonomous_dripper.py

```diff
@@ -0,0 +1,145 @@
+from datetime import datetime
+
+import brownie
+import pytest
+from brownie import accounts, chain, interface
+from brownie_tokens import MintableForkToken
+
+from helpers.addresses import registry
+
+
+@pytest.fixture(scope='module')
+def deployer():
+    return accounts[0]
+
+
+@pytest.fixture(scope='module')
+def dripper():
+    from scripts.deployment.deploy_autonomous_dripper import main
```

Comment:
Love that you use the real deployment script within the test. Good stuff. I am going to have to loose my fear of deploying/verifying from brownie, if you say you have been successful from Brownie's latest it will make the deploying process a lot easier in the future. 

I wonder how you handle gas strategies and setting the priority fee. I couldn't find anywhere in the script where you do this programmatically.

Reply:
https://eth-brownie.readthedocs.io/en/stable/core-contracts.html#transaction-parameters

Reply:
https://eth-brownie.readthedocs.io/en/stable/core-gas.html

Reply:
You should look into Andrii's work on the `bader-utils` for the customizable gas-strategies: https://github.com/Badger-Finance/badger-utils#using-gas_utils-module